dummyapp/lib/tasks/annotate_rb.rake

```diff
@@ -0,0 +1,8 @@
+# This rake task was added by annotate_rb gem.
+
+# Can set `ANNOTATERB_SKIP_ON_DB_TASKS` to be anything to skip this
+if Rails.env.development? && ENV["ANNOTATERB_SKIP_ON_DB_TASKS"].nil?
```

Comment:
Minor: but I might have gone with:

```ruby
if Rails.env.development? && !ENV["ANNOTATERB_SKIP_ON_DB_TASKS"]
```

This is because you could set this environment variable to `false` and it would skip annotations, but probably not that big of a deal.

Reply:
I don't know if it's shell/environment specific but I _think_ any values are interpreted as strings and not parsed.

```ruby
# $ A_STRING=false pry

[1] pry(main)> ENV['A_STRING']
=> "false"
```

Reply:
That's true, I guess that makes my comment not as relevant. (Also realized I had the boolean logic backwards and had included a `!`)

Reply:
No worries. I wasn't sure how ENV behaved, so when I first read your comment I thought this could be changed as well.